From: Simon McVittie <smcv@collabora.com>
Date: Tue, 17 Dec 2019 10:32:45 +0000
Subject: flatpak.supp: Silence GFile leak in
 flatpak_get_user_base_dir_location()

This is deliberately allocated once per process and never freed.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: https://github.com/flatpak/flatpak/pull/3312
---
 tests/flatpak.supp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/tests/flatpak.supp b/tests/flatpak.supp
index b5207cc..4736b87 100644
--- a/tests/flatpak.supp
+++ b/tests/flatpak.supp
@@ -237,3 +237,11 @@
    fun:gnutls_x509_ext_import_subject_alt_names
    fun:gnutls_x509_crt_import
 }
+# Deliberately leaking once per process
+{
+   flatpak_get_user_base_dir_location
+   Memcheck:Leak
+   ...
+   fun:g_file_new_for_path
+   fun:flatpak_get_user_base_dir_location
+}
